code/code/cmd/cmd_bash.cc

```diff
@@ -62,15 +63,11 @@ bool TBeing::canBash(TBeing *victim, silentTypeT silent)
     default:
       break;
   }
-  if (desc && (isFlying() && !victim->isFlying()))  {
-    if (!silent)
-      sendTo("You can't bash while flying if your victim isn't flying.\n\r");
-    return FALSE;
-  }
+
   if (checkPeaceful("You feel too peaceful to contemplate violence.\n\r"))
     return FALSE;
 
-  if (eitherLegHurt()) {
+  if (eitherLegHurt() && !isFlying()) {
```

Comment:
Figured you should still be able to bash into someone even if you have a damaged leg if you're flying